lib/opentelemetry/exporters/datadog/version.rb

```diff
@@ -9,7 +9,7 @@ module OpenTelemetry
   module Exporters
     module Datadog
       ## Current OpenTelemetry Datadog exporter version
-      VERSION = '0.2.0'
+      VERSION = '0.3.0'
```

Comment:
I suggest going for 0.2.1, since functionally nothing changed. (Technically, under semantic versioning anything before 1.0.0 is fair game, but I think it makes sense to try to keep to it if we can)

```suggestion
      VERSION = '0.2.1'
```